runtime/queries/clojure/highlights.scm

```diff
@@ -0,0 +1,71 @@
+(dis_expr) @comment
+
+(kwd_lit) @type
+
+(str_lit) @string
+
+(num_lit) @constant.numeric
+
+[(bool_lit) (nil_lit)] @constant.builtin
+
+(comment) @comment
+
+(regex_lit) @string.special
```

Comment:
We could use `@string.regexp` for this if you like. Or we could have an injection for tree-sitter-regex:

```scm
; runtime/queries/clojure/injections.scm
((regex_lit) @injection.content
 (#set! injection.language "regex"))
```